aws/modules/consolidate-logs/module.ftl

```diff
@@ -0,0 +1,197 @@
+                                "Functions": {
+                                    "processor": {
+                                        "RunTime": "python3.6",
+                                        "MemorySize": 128,
+                                        "Timeout": 30,
+                                        "Handler": "src/run.lambda_handler",
+                                        "Links": {
+                                            "feed" : {
+                                                "Tier" : tier,
+                                                "Component" : datafeedName,
+                                                "Instance" : "",
+                                                "Version" : "",
+                                                "Role" : "logwatch",
+                                                "Direction": "inbound"
+                                            }
+                                        }
+                                    }
+                                }
+                            }
+                        }
+                    }
+                }
+            },
+            "LogFilters": {
+                "all-logs": {
+                    "Pattern": ""
+                }
+            },
+            "LoggingProfiles" : {
+                "consolidate" : {
+                    "ForwardingRules": {
+                        "store": {
+                            "Filter": "all-logs",
+                            "Links": {
+                                "feed": {
+                                    "Tier": tier,
+                                    "Component": datafeedName,
+                                    "Instance": "",
+                                    "Version": ""
+                                }
+                            }
+                        }
+                    }
+                }
+            },
+            "DeploymentProfiles" : {
+                deploymentProfile : {
+                    "Modes" : {
+                        "*" : {
+                            "*" : {
+                                "Profiles" : {
+                                    "Logging" : "consolidate"
+                                }
+                            },
+                            "apigateway" : {
+                                "AccessLogging" : {
+                                    "Enabled" : true,
+                                    "aws:KinesisFirehose" : true,
+                                    "aws:KeepLogGroup" : true
+                                },
+                                "WAF" : {
+                                    "Logging" : {
+                                        "Enabled" : true
+                                    }
+                                }
+                            },
+                            "cdn" : {
+                                "EnableLogging" : true
```

Comment:
Do we still need the WAF Logging true on the cdn - it may be the default but worth being explicit

Reply:
Fixed up